lib/ReactViewModels/MouseCoords.ts

```diff
@@ -37,15 +48,18 @@ export default class MouseCoords {
     Cancelable;
   tileRequestInFlight?: unknown;
 
-  @observable elevation?: string;
-  @observable utmZone?: unknown;
-  @observable latitude?: string;
-  @observable longitude?: string;
-  @observable north?: unknown;
-  @observable east?: unknown;
-  @observable cartographic?: Cartographic;
```

Comment:
From a performance perspective, it is perhaps a wrong idea to use `@observable` for very frequently updated values because of the mobx overhead. We instead introduce a new `updateEvent` to subscribe to changes.